packages/issuer/lib/VcIssuer.ts

```diff
@@ -24,13 +26,16 @@ import {
   TYP_ERROR,
 } from '@sphereon/openid4vci-common'
 import { ICredential, W3CVerifiableCredential } from '@sphereon/ssi-types'
+import { v4 } from 'uuid'
 
 export class VcIssuer {
   _issuerMetadata: IssuerMetadata
   _userPinRequired?: boolean
   _issuerCallback?: CredentialIssuerCallback
   _verifyCallback?: JWTVerifyCallback
   private readonly _stateManager: ICredentialOfferStateManager
+  private readonly _cNonce: string[] = []
+  private readonly _cNonceExpiresIn: number = parseInt(process.env.C_NONCE_EXPIRES_IN as string) * 1000 || 90 * 1000
```

Comment:
Hmzz, add a todo that we also want some config option in the future next to env vars

Reply:
Fixed!